### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by varyvol**
Although the option to enable timestamps for all Pipeline builds was already explained, it now mentions where it can be changed and also a screenshot is displayed.

@basil @jglick

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ To enable timestamps for multiple builds at once, use the [Configuration Slicin
 
 **Since Timestamper 1.9**
 
-Set the global option to enable timestamps for all Pipeline builds, or use the step as described below.
+Set the global option to enable timestamps for all Pipeline builds (in "Manage Jenkins" -> "Configure"), or use the step as described below.
```
Comment (MRamonLeon) [inline README.md]: would you like?
```suggestion
Set the global option to enable timestamps for all Pipeline builds (in _**Manage Jenkins**_ -> _**Configure**_), or use the step as described below.
```

**Comment by basil** [comment]
I don't seem to have write permissions to be able to merge this, but presumably Jesse can take care of it.

**Comment by varyvol** [comment]
@jglick if you could please merge this and possibly release together with some pending stuff in master, that would be great.

**Comment by oleg-nenashev** [comment]
@basil You should have the permissions now

**Comment by oleg-nenashev** [comment]
FTR https://github.com/jenkins-infra/repository-permissions-updater/blob/master/permissions/plugin-timestamper.yml and https://github.com/jenkins-infra/repository-permissions-updater/pull/1308 for the justification.

**Comment by jglick** [comment]
Indeed I could be the maintainer of last resort here, but I do not really have the time to give it the proper level of attention.

**Comment by varyvol** [comment]
@basil could you generate a new release including this and other changes in master?

**Comment by basil** [comment]
> @basil could you generate a new release including this and other changes in master?

I have just released [Timestamper 1.11](https://github.com/jenkinsci/timestamper-plugin/releases/tag/timestamper-1.11).

**Comment by varyvol** [comment]
@basil awesome, thanks!

**Pull request merged**